pull.go

```diff
@@ -171,7 +173,21 @@ func pullImage(ctx context.Context, store storage.Store, imageName string, optio
 	if err == nil {
 		return destRef, nil
 	}
-	return nil, err
+
+	// If no image was found, we should handle.  Lets be nicer to the user and see if we can figure out why.
+	registryPath := sysregistries.RegistriesConfPath(&types.SystemContext{})
+	searchRegistries, err := registries.GetRegistries()
+	if err != nil {
+		return nil, err
```

Comment:
Does this fail if /etc/containers/registies.conf does not exist?

Reply:
Yes, it throws:

```
unable to parse the registries.conf file: open /etc/containers/registries.conf: no such file or directory
```